Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -95,7 +95,7 @@ class RequestArguments:
     FILTER_FIELDS_ON_FORMAT_JSON = "name,type"
 
     def __init__(self,
-                 query: str,
+                 query: str = '',
```

Comment:
Why do we think this is the problem? we are setting by default empty string 
https://github.com/demisto/content/blob/aa9fe55047aca22fddf83a62ab5f13b4ad63d3c1/Packs/EDL/Integrations/EDL/EDL.py#L833

Reply:
here is the problem when we are getting the data from the integration_context we use assign_params which removes the query if it is an empty string, and on creating with this data a new RequestArguments object the query is None (because there is not a default value for the query on the init of the class).
https://github.com/demisto/content/blob/aa9fe55047aca22fddf83a62ab5f13b4ad63d3c1/Packs/EDL/Integrations/EDL/EDL.py#L160-L162

Reply:
I see thanks